.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+Fixes #ISSUE.
+
+
+## Pre-merge checklist
+
+<!--- Docs: A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
+
+- [ ] Docs
+   - `docs/` have been added/updated **OR**
+   - No updates to the docs necessary after careful consideration.
+
+- [ ] QC
+   - `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
+   -  No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
+
+- [ ] Account for any new packages
+  - Open an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) to request addition. Can also open a PR. Python package changes should occur in [[requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full)](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full), and non-Python packages in [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile).
+
+- [ ] Reviewed
+   - Has been sufficiently reviewed by at least one review from a different team member of the Mondo Technical team.
+
+## Overview
```

Comment:
```suggestion
## Overview
<!--- Short paragraph or short bulleted list describing changes -->
```
I tried to select multiple lines to suggest replacement of multiple lines ([step 4 in this guide from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)), but it didn't work.

Reply:
The overview should be a few sentences that describes what problem is being solved with this PR. It's fine to refer back to the issue for more detailed information, but the comment in the PR should be clear enough to understand the problem. 

The Overview should also point out what the major change was to solve the problem, e.g. fixed bug in method xyz or updated the make statement for goal 123.

The Overview should also point out what is the expected result of the change of the code fix, e.g. when `sh run.sh make build-mondo-ingest` is run on the branch what is the new behavior.

Reply:
Yea agreed with that @twhetzel. @joeflack4 can fold this into your suggestion above?

Reply:
> The Overview should also point out what the major change was to solve the problem, e.g. fixed bug in method xyz or updated the make statement for goal 123.

I agree with that when it is applicable. Just spitballing, I think PRs have been small enough for single changes like these 25% of the time or so.

Reply:
Alright done, feel free to commit the suggestion if you guys like it!